addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -79,28 +79,31 @@
     .o_survey_kanban_card_ungrouped {
         .ribbon {
             // Desktop: finishes on next kanban card line
-            height: 77px;
-            width: 130px;
+            height: 100px;
+            width: 125px;
 
             // Mobile: is in a corner, takes more place
             @include media-breakpoint-down(md) {
                 height: 100px;
-                width: 100px;
+                width: 125px;
             }
-
+            
```

Comment:
:socks: extra spaces to remove ?

Reply:
Corrected and finally all the checks are also greened :partying_face:

Reply:
Also, Should I mention in commit that it should be only deployed from 16.0 to 16.2? (there is no issue in new design: 16.3 and up )